packages/std/src/errors/backtrace.rs

```diff
@@ -0,0 +1,48 @@
+use core::fmt::{Debug, Display, Formatter, Result};
+use thiserror::Error;
+
+/// This wraps an actual backtrace to achieve two things:
+/// - being able to fill this with a stub implementation in `no_std` environments
+/// - being able to use this in conjunction with [`thiserror::Error`]
+#[derive(Error)]
+pub struct BT(Box<dyn Printable>);
+
+impl BT {
+    #[track_caller]
+    pub fn capture() -> Self {
+        BT(Box::new(std::backtrace::Backtrace::capture()))
```

Comment:
okay, great. If we know it is always disabled, fine

Reply:
It will always be the case because the check for the env variable always errors on wasm32-unknown-unknown and I don't see any other reasonable behaviour they could implement there. So I suspect that whole env var checking to be optimized away.
That being said, I compiled contracts with both versions and there does still seem to be a size difference in the final wasm, so I will add your suggestion.